documentation/docs/examples/clipping.mdx

```diff
@@ -0,0 +1,79 @@
+describe: Reveal support adding a number of slicing planes to "cut through" the models.
+---
+
+import { DemoWrapper } from '@site/src/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Slicing planes, also called clipping planes, can be used to "cut through" the model to reveal the interior. This is 
+useful to isolate interesting parts of the model and hide obscuring geometry. Reveal supports having multiple slicing planes
+in arbitrary position and orientation. When slicing planes are active,
+the amount of geometry loaded might be reduced since culled geometry doesn't need
+to be loaded (or rather loaded in a lower detail level).
+
+<video preload="auto" autoPlay="autoplay" loop controls>
+  <source src={useBaseUrl('/img/cad-clipping.webm')} />
+</video>
+
+:::warning
+Clipping planes are currently not supported by point cloud models.
+:::
+
+## Examples
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+### Single, fixed slicing plane
+
+Slicing planes are defined by providing zero or more [`THREE.Plane`](https://threejs.org/docs/#api/en/math/Plane) 
+to `Cognite3DViewer.setSlicingPlanes`. The slicing planes are activated for all models and
+replaces any active slicing planes.
+
+Typically, [setFromNormalAndCoplanarPoint](https://threejs.org/docs/#api/en/math/Plane.setFromNormalAndCoplanarPoint)
+is used to create slicing planes. This function accepts a direction that defines the orientation of
+the plane and a point on the plane. For example will the following
+code create a slicing plane that only accepts geometry below `Y = 55`:
+```jsx runnable
+const orientation = new THREE.Vector3(0, -1, 0);
+const point = new THREE.Vector3(0, 55, 0);
+const plane = new THREE.Plane().setFromNormalAndCoplanarPoint(orientation, point);
+viewer.setSlicingPlanes([plane]);
+```
+
+### Animated slicing plane
+
+Planes can be manipulated runtime and effects will have effect in the viewer. Note however that rendering must be 
+triggered after updating the plane before the change will have visual effect. This can be done using 
+`Cognite3DViewer.forceRerender()`.
+
+Below is an example animates a slicing plane by ping-ponging the plane in the vertical direction.
+
+```js runnable
+const bbox = model.getModelBoundingBox();
+
+const orientation = new THREE.Vector3(0, -1, 0);
+const point = new THREE.Vector3(0, bbox.max.y, 0);
+const plane = new THREE.Plane();
+viewer.setSlicingPlanes([plane]);
+
+const start = performance.now();
+setInterval(() => {
+  // Ping-pong plane between top of model and half-way to the bottom
+  const dt = (performance.now() - start) / 1000.0
+  const s = 0.5 * (Math.sin(Math.PI * dt / 10.0) + 1.0);
+  point.y = bbox.max.y - (bbox.max.y - bbox.min.y) / 2.0 * s;
+  plane.setFromNormalAndCoplanarPoint(orientation, point);
+
+  viewer.forceRerender();
+}, 1000 / 60);
+```
+:::tip
+Updating `THREE.Plane`-instances will not trigger data loading. Typically you will need to trigger data loading if
```

Comment:
I read this text multiple times to understand it, perhaps we just need some proper example here? I mean, the one where you do things right as you described here:

> Typically you will need to trigger data loading if you do substantial changes to the slicing planes.

Because currently the example looks cool, but it leaves me without knowledge on how to make things right. And it requires some patience to understand what is written in that tip, but basically, the tip just says - "hey, the example looks cool, but don't do that in real code, do X instead". So maybe we can have the X as an additional example here?

Reply:
Just realized that "the X" example is the first one after checking the docs preview 🙄

Reply:
Updated a bit - hopefully more clear now :)